composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
>If some third party transport breaks in future version (like Monolog) and so our code breaks our user will not want to hear explanations like "this is not our code broken"

Then it means you don't trust that a vendor already tests his product

>until it means that we must to install ftp-servers, email-servers etc.

Then you have no idea of what unit testing means